stores/s3vectors/package.json

```
@@ -46,6 +46,6 @@
     "@internal/types-builder": "workspace:*"
   },
   "peerDependencies": {
-    "@mastra/core": ">=0.10.7-0 <0.14.0-0"
+    "@mastra/core": ">=0.15.3-0 <0.16.0-0"
```

I think you'll want to change this to be:

```suggestion
    "@mastra/core": ">=0.15.3-0 <0.17.0-0"
```